pkg/services/live/pushws/ws.go

```diff
@@ -87,21 +87,24 @@ func setupWSConn(ctx context.Context, conn *websocket.Conn, config Config) {
 	}
 	if pingInterval > 0 {
 		pongWait := pingInterval * 10 / 9
-		_ = conn.SetReadDeadline(time.Now().Add(pongWait))
-		conn.SetPongHandler(func(string) error {
+		// _ = conn.SetReadDeadline(time.Now().Add(pongWait))
+		conn.SetPongHandler(func(v string) error {
+			fmt.Printf("PONG %s\n", v)
 			_ = conn.SetReadDeadline(time.Now().Add(pongWait))
 			return nil
 		})
 	}
 
 	go func() {
-		ticker := time.NewTicker(25 * time.Second)
```

Comment:
is the 25 intentional? or should it match the ping interval (defaults to 25s)

Reply:
Should be `pingInterval` instead of `25 * time.Second`.